src/round.js

```diff
@@ -0,0 +1,40 @@
+class Round {
+  constructor(players, data, puzzleBank) {
+    this.players = players;
```

Comment:
I would question whether or not you truly need to save your players on your round class as an instance property.....how many methods are using `this.players`? Could those methods instead take in the player instance as an argument where they're invoked?